cloudinit-updates-jenkins-io.tftpl

```diff
@@ -0,0 +1,19 @@
+#cloud-config
+write_files:
+  - path: /run/puppetinstall/puppet.conf
```

Comment:
Can you comment why do you write at this path (and why later you have to move this file) ?

Reply:
comments added to code, in case you mean here : 
I cannot put the puppet agent config file as it fail the package installation (even with --yes) so I create a temporary file and move it at the final path : /run/puppetinstall/puppet.conf once the package installed.

Reply:
I would add that I did not use the /tmp dir because : 
`when writing files, do not use /tmp dir as it races with systemd-tmpfiles-clean LP: #1707222. Use /run/somedir instead.` : https://cloudinit.readthedocs.io/en/latest/topics/modules.html#runcmd